src/worker.cc

```diff
@@ -730,7 +730,7 @@ void from_json(rapidjson::Document& doc, Options& options) {
     options.set_height_precision(*height_precision);
   }
 
-  options.set_verbose(rapidjson::get(doc, "/verbose", false));
+  options.set_verbose(rapidjson::get<bool>(doc, "/verbose", false));
```

Comment:
hmm are there any side effects of this? i forget exactly but i put a lot of handling for allowing conversions between different types (especially primitive ones). does verbose=1 still work? the reason i did this was to maintain api compatibility for back when we were using ptree to parse json. it is very loose in this way and will convert integers and bools freely

Reply:
hm.. that seems like a artifact on my side